src/combinatorics/simple_graph/subgraph.lean

```diff
@@ -348,10 +354,17 @@ def finite_at_of_subgraph {G' G'' : subgraph G} [decidable_rel G'.adj]
    fintype (G'.neighbor_set v) :=
 set.fintype_subset (G''.neighbor_set v) (neighbor_set_subset_of_subgraph h v)
 
+noncomputable instance (G' : subgraph G) [fintype G'.verts] (v : V) : fintype (G'.neighbor_set v) :=
```

Comment:
It's not so clear to me that we want this instance. The problem is that if we use locally finite subgraphs lemmas on finite subgraphs, the latter will have the above `fintype (G'.neighbor_set v)` instance which might not match the random `fintype (G'.neighbor_set v)` instance from the more general lemmas.

Reply:
If we're careful (as we should be) to always have lemmas that concern local finiteness at a vertex take a `fintype (G'.neighbor_set v)` instance, we should be fine, unless you're meaning something else here.

What we are definitely missing here, though, is a computable version of this instance, which likely ought to replace this one since in a `classical` setting there will be enough decidable instances to trigger it.

Reply:
Yes, I agree on both points. What I'm foreseeing is that one day someone will add another construction of the same instance because "all lemmas that concern local finiteness take a `fintype (G'.neighbor_set v)` so what could possibly go wrong?" and then the instances conflict because they are doing the same thing in different ways. So this is a future problem, not a present one, and it is currently fine as is.